src/afl-fuzz-run.c

```diff
@@ -897,11 +910,10 @@ u8 trim_case(afl_state_t *afl, struct queue_entry *q, u8 *in_buf) {
 
     }
 
-    if (custom_trimmed) return trimmed_case;
+    if (custom_trimmed) { fault = trimmed_case; goto abort_trimming_wo_count; }
```

Comment:
I'm not sure this was intentional, but the old code path didn't log the trimed size (same for inputs too small to trim). I opted to keep the original behavior, even though I suspect in this case (but not the other), that's not actually intended behavior.

Reply:
what do you mean by "didn't log the trimed size"? the return of trim_case is only to check for forkserver errors.